### ISSUE TITLE
If the first imported line start with an indentation, the indentation is stripped

### ISSUE THREAD

**Issue by MangelMaxime** [kevin940726/remark-code-import#14]
Hello,

I am importing the file content via `./../demo/src/Demo.fs#L15-L16`

Which correspond to these lines in my files:

```fs
    Tippy.create [
        Tippy.plugins [|
```

which generates:

![image](https://user-images.githubusercontent.com/4760796/140545504-7b7a1984-4706-4c55-ae35-2f372ab00b01.png)

As you can see, there is a "huge" indentation on the second line because the indentation on the first line has been stripped I believe.

If I use a first line without indentation, I get the expected result:

```fs
let Component () =
    Tippy.create [
        Tippy.plugins [|
```

gives:

![image](https://user-images.githubusercontent.com/4760796/140545695-0606939c-e821-460c-9a22-56d676f05a80.png)

**Comment by kevin940726** [comment]
Good catch!

I just published `v0.4.0` to fix this issue and include two additional options (`preserveTrailingNewline` and `removeRedundantIndentations`). Please see README for usages if you need them.

Thanks again!

**Comment by MangelMaxime** [comment]
Thank you for the quick fix.

I think `removeRedundantIndentations` is a really interesting option for some of my use cases.
